### ISSUE TITLE
Failed to load any font ?

### ISSUE THREAD

**Issue by cournape** [enthought/enable#266]
The following w/ enable 4.6.0 on a basic centos 6.5 system:

``` bash
$ edm run -e build-env -- python -c "from kiva.fonttools import afm"                                                                                                                                              
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/vagrant/.edm-build/envs/build-env/lib/python2.7/site-packages/kiva/__init__.py", line 21, in <module>
    from .fonttools import Font
  File "/home/vagrant/.edm-build/envs/build-env/lib/python2.7/site-packages/kiva/fonttools/__init__.py", line 1, in <module>
    from .font import Font, str_to_font
  File "/home/vagrant/.edm-build/envs/build-env/lib/python2.7/site-packages/kiva/fonttools/font.py", line 11, in <module>
    from .font_manager import FontProperties, fontManager
  File "/home/vagrant/.edm-build/envs/build-env/lib/python2.7/site-packages/kiva/fonttools/font_manager.py", line 1418, in <module>
    _rebuild()
  File "/home/vagrant/.edm-build/envs/build-env/lib/python2.7/site-packages/kiva/fonttools/font_manager.py", line 1367, in _rebuild
    fontManager = FontManager()
  File "/home/vagrant/.edm-build/envs/build-env/lib/python2.7/site-packages/kiva/fonttools/font_manager.py", line 1078, in __init__
    self.defaultFont['ttf'] = self.ttffiles[0]
IndexError: list index out of range
```

**Comment by corranwebster** [comment]
Even if there are no fonts installed on the system, we should fail more gracefully than this. In particular, if we can do it on use, rather than import, that would be better.

**Comment by corranwebster** [comment]
Also, this may be moot with #264, since that replaces all the fonttools stuff with a third-party package.

**Comment by cournape** [comment]
I tried enable master, and got exactly the same error FWIW.

**Comment by andrewcollette** [comment]
FWIW, with enable 4.5.1 it prints the following lines and then proceeds normally, and creates a "kiva/fontList.cache" file in the preferences dir of my Canopy Data app.  On the next launch, the lines aren't printed and the app launches normally.  This is all using EDM on OS X.  With enable 4.6.0 I get the error described in this issue.
```
Fontconfig error: Cannot load default config file
Fontconfig error: Cannot load default config file
```

**Comment by mdickinson** [comment]
Adding the BUG label, since this seems to be a regression since the previous release of Enable.

**Comment by mdickinson** [comment]
This was fixed in master #272.
